docs/installation.rst

```diff
@@ -25,7 +25,7 @@ INSTALLED_APPS
 
    INSTALLED_APPS = (
        ...
-       'django_nyt'
```

Comment:
Is this part necessary, though? Because `default_app_config` is already specified?

Reply:
From docs https://docs.djangoproject.com/en/2.0/ref/applications/#configuring-applications

`New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.`

I'm not aware when/if `default_app_config` will be deprecated/removed.

Reply:
Okay, thanks for the refence -- that was actually what I read and didn't really understand... in the commit message, it seems that @aaugustin indicates with this that there is a reason, it's just not evident to me:

> but at least the reasons for avoiding it will be documented.

(from https://github.com/django/django/commit/94a36cfd8c66a6be9a1e001b29e4c55071f4e0de -- which was merged for Django 1.9 btw)

I would argue that most applications have exactly 1 default AppConfig (and not others), and that `INSTALLED_APPS` can explicitly use the *default* AppConfig, and then the application can explicitly specify the *default* AppConfig :)

Reply:
Probably yes it is a little annoying, but I think having it explicit it is better in case someone needs to override it.
By the way if I can I cut and paste from docs, so having the longer or the short name make really no difference.

Tell me what to do if use the dotted path or keep the short one.

Reply:
My main objection is to make everyone dance around a weakly argued necessity for change.. "disruption for the sake of disruption"... L'art Pour L'art... the Sisyphus version :)

No, don't change it, it's good to be aligned with the recommendation from the docs. And thanks for noticing!